src/lib/react/useRoomConnection.ts

```diff
@@ -11,10 +11,12 @@ import RoomConnection, {
 } from "../RoomConnection";
 import { LocalParticipantState, RemoteParticipantState, Screenshare, WaitingParticipant } from "../RoomParticipant";
 
+type LocalScreenshareStatus = undefined | "starting" | "active";
```

Comment:
how do we feel about moving the optional-ness to the object property like
```typescript
type LocalScreenshareStatus = "starting" | "active"
export interface RoomConnectionState {
    ...rest;
    localScreenshareStatus?: LocalScreenshareStatus;
}
```

Reply:
I dont have strong opinions here, but I see we are doing the same thing with recording and streaming state. If there is no state, the property is undefined